frappe/automation/workspace/tools/tools.json

```diff
@@ -208,16 +208,17 @@
    "type": "Link"
   }
  ],
- "modified": "2022-01-13 17:48:48.456763",
+ "modified": "2022-08-02 10:52:01.004944",
  "modified_by": "Administrator",
  "module": "Automation",
  "name": "Tools",
  "owner": "Administrator",
  "parent_page": "",
  "public": 1,
+ "quick_lists": [],
  "restrict_to_domain": "",
  "roles": [],
- "sequence_id": 26.0,
+ "sequence_id": 20.0,
```

Comment:
@1ubuntuuser can you undo all workspace JSON file changes? I think you accidentally updated the standard workspaces. 
tools.json, build.json, settings.json, users.json, customization.json, integrations.json, website.json

Reply:
Thanks, Will do.

Reply:
I think you missed tools, integrations and website

Reply:
Thanks, I'll update those and the unused vars.